### PR TITLE
Described IRIs as "resolved" rather than "absolute".

### PR DESCRIPTION
**Pull request by gkellogg**
For #15.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/pull/58.html" title="Last updated on Aug 21, 2023, 4:11 PM UTC (5170cd5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/58/664a651...5170cd5.html" title="Last updated on Aug 21, 2023, 4:11 PM UTC (5170cd5)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
@afs, I think this addresses your concern about the mis-use of "absolute IRI", at least within this document. If the general direction seems reasonable, I can replicate to other specifications.

File: spec/index.html [hunk 1]
```diff
@@ -599,11 +600,11 @@ <h3>IRIs</h3>
         IRIs are a generalization of
         <dfn data-lt="URI" data-lt-noDefault><abbr title="Uniform Resource Identifier">URI</abbr>s</dfn>
         [[RFC3986]] that permits a wider range of Unicode characters.
-        Every absolute URI and URL is an IRI, but not every IRI is an URI.
+        Every URI and URL is an IRI, but not every IRI is an URI.
```
Comment (TallTed) [inline spec/index.html:603]:
```suggestion
        Every URI and URL is an IRI, but not every IRI is a URI.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
a yew-are-eye, a yew-are-ell, an eye-are-eye... :-)

File: spec/index.html [hunk 2]
```diff
@@ -582,15 +582,16 @@ <h3>IRIs</h3>
...
+      per [[RFC3986]],
       and MAY contain a fragment identifier.</p>
 
     <p><dfn>IRI equality</dfn>:
       Two IRIs are the same if and only if they compare as identical when
       considered as character strings, as in Simple String Comparison in
       <a data-cite="rfc3987#section-5.3.1">section 5.3.1</a>
       of [[!RFC3987]].
-      (This is done in the abstract syntax, so the IRIs are absolute
+      (This is done in the abstract syntax, so the IRIs are fully resolved
```
Comment (afs) [inline spec/index.html:594]:
```suggestion
      (This is done in the abstract syntax, so the IRIs are resolved
```
I don't think there is a "partially resolved".

**Comment by afs** [comment]
> @afs, I think this addresses your concern about the mis-use of "absolute IRI", at least within this document. If the general direction seems reasonable, I can replicate to other specifications.

I can't think of a better name at the moment. For other docs, just "IRI" is reasonable. The syntax parsing step will cover the need to make complete. We can overload text about details and harm readability by always putting in our local terminology.

Going back to 3986/3987: I can't find terminology for:
```
URI         = scheme ":" hier-part [ "?" query ] [ "#" fragment ]
```
with 
```
hier-part   = "//" authority path-abempty
```
but none of the other `hier-part` alternatives.

File: spec/index.html [hunk 3]
```diff
@@ -599,11 +600,11 @@ <h3>IRIs</h3>
         IRIs are a generalization of
         <dfn data-lt="URI" data-lt-noDefault><abbr title="Uniform Resource Identifier">URI</abbr>s</dfn>
         [[RFC3986]] that permits a wider range of Unicode characters.
-        Every absolute URI and URL is an IRI, but not every IRI is an URI.
+        Every URI and URL is an IRI, but not every IRI is a URI.
         In RDF, IRIs are used as <dfn data-lt="iri reference">IRI references</dfn>, as defined in [[RFC3987]]
         <a data-cite="RFC3987#section-1.3">section 1.3</a>.
         An IRI reference is common usage of an Internationalized Resource Identifier.
-        An IRI reference refers to either <a>IRI</a> or <a>relative IRI reference</a>,
+        An IRI reference refers to either a fully resolved <a>IRI</a> or <a>relative IRI reference</a>,
```
Comment (afs) [inline spec/index.html:607]:
```suggestion
        An IRI reference refers to either a resolved <a>IRI</a> or <a>relative IRI reference</a>,
```

File: spec/index.html [hunk 4]
```diff
@@ -1107,7 +1108,7 @@ <h3>The XML Schema Built-in Datatypes</h3>
       <tr><td><a data-cite="xmlschema11-2#base64Binary"><code>xsd:base64Binary</code></a></td><td>Base64-encoded binary data</td></tr>
 
       <tr><th rowspan="7">Miscellaneous<br />XSD types</th>
-          <td><a data-cite="xmlschema11-2#anyURI"><code>xsd:anyURI</code></a></td><td>Absolute or relative URIs and IRIs</td></tr>
+          <td><a data-cite="xmlschema11-2#anyURI"><code>xsd:anyURI</code></a></td><td>Fully resolved or relative URI and IRI references</td></tr>
```
Comment (afs) [inline spec/index.html:1111]:
```suggestion
          <td><a data-cite="xmlschema11-2#anyURI"><code>xsd:anyURI</code></a></td><td>Resolved or relative URI and IRI references</td></tr>
```

File: spec/index.html [hunk 5]
```diff
@@ -1644,6 +1645,11 @@ <h2>Changes between RDF 1.1 and RDF 1.2</h2>
       for informative definition of a <a>quad</a>.</li>
     <li>Added <a href="#section-quoted-triples" class="sectionRef"></a>
       and definitions for <a>quoted triple</a> and <a>asserted triple</a>.</li>
+    <li>Improved the use of IRI terminology,
+      and added <a href="#iri-abnf" class="sectionRef"></a>.
+      This improves the language using <a>relative IRI references</a>
+      and clarifies that, in the abstract syntax, IRIs are fully resolved,
```
Comment (afs) [inline spec/index.html:1651]:
```suggestion
      and clarifies that, in the abstract syntax, IRIs are resolved,
```

File: spec/index.html [hunk 6]
```diff
@@ -582,15 +582,16 @@ <h3>IRIs</h3>
     <p class="note">For convenience, a complete [[ABNF]] grammar
       from [[RFC3987]] is provided in <a href="#iri-abnf" class="sectionRef"></a>.</p>
 
-    <p>IRIs in the RDF abstract syntax MUST be absolute,
+    <p>IRIs in the RDF abstract syntax MUST be <a data-cite="RFC3986#section-5">fully resolved</a>
```
Comment (gkellogg) [inline spec/index.html:585]:
```suggestion
    <p>IRIs in the RDF abstract syntax MUST be <a data-cite="RFC3986#section-5">resolved</a>
```

**Review by hartig — APPROVED** [review]

**Review by afs — APPROVED** [review]

**Pull request merged**